cmd/dcld/cmd/light.go

```diff
@@ -75,37 +95,43 @@ var (
 
 	primaryKey   = []byte("primary")
 	witnessesKey = []byte("witnesses")
+
+	startTimeout int64
 )
 
 func init() {
-	LightCmd.Flags().StringVar(&listenAddr, "laddr", "tcp://localhost:8888",
+	LightCmd.Flags().StringVar(&listenAddr, FlagListenAddr, "tcp://0.0.0.0:8888",
 		"serve the proxy on the given address")
-	LightCmd.Flags().StringVarP(&primaryAddr, "primary", "p", "",
+	LightCmd.Flags().StringVarP(&primaryAddr, FlagPrimary, FlagPrimaryShort, "",
 		"connect to a Tendermint node at this address")
-	LightCmd.Flags().StringVarP(&witnessAddrsJoined, "witnesses", "w", "",
+	LightCmd.Flags().StringVarP(&witnessAddrsJoined, FlagWitness, FlagWitnessShort, "",
 		"tendermint nodes to cross-check the primary node, comma-separated")
-	LightCmd.Flags().StringVarP(&dir, "dir", "d", os.ExpandEnv(filepath.Join("$HOME", ".tendermint-light")),
+	LightCmd.Flags().StringVarP(&dir, FlagDir, FlagDirShort, os.ExpandEnv(filepath.Join("$HOME", ".tendermint-light")),
 		"specify the directory")
 	LightCmd.Flags().IntVar(
 		&maxOpenConnections,
-		"max-open-connections",
+		FlagMaxConn,
 		900,
 		"maximum number of simultaneous connections (including WebSocket).")
-	LightCmd.Flags().DurationVar(&trustingPeriod, "trusting-period", 168*time.Hour,
+	LightCmd.Flags().DurationVar(&trustingPeriod, FlagTrustPeriod, 168*time.Hour,
 		"trusting period that headers can be verified within. Should be significantly less than the unbonding period")
-	LightCmd.Flags().Int64Var(&trustedHeight, "height", 1, "Trusted header's height")
-	LightCmd.Flags().BytesHexVar(&trustedHash, "hash", []byte{}, "Trusted header's hash")
-	LightCmd.Flags().StringVar(&logLevel, "log-level", "info", "The logging level (debug|info|warn|error|fatal)")
+	LightCmd.Flags().Int64Var(&trustedHeight, FlagHeight, 0, "Trusted header's height")
+	LightCmd.Flags().BytesHexVar(&trustedHash, FlagHash, []byte{}, "Trusted header's hash")
+	LightCmd.Flags().StringVar(&logLevel, FlagLogLevel, "info", "The logging level (debug|info|warn|error|fatal)")
 	// LightCmd.Flags().StringVar(&logFormat, "log-format", log.LogFormatPlain, "The logging format (text|json)")
-	LightCmd.Flags().StringVar(&trustLevelStr, "trust-level", "1/3",
+	LightCmd.Flags().StringVar(&trustLevelStr, FlagTrustLevel, "1/3",
 		"trust level. Must be between 1/3 and 3/3",
 	)
-	LightCmd.Flags().BoolVar(&sequential, "sequential", false,
+	LightCmd.Flags().BoolVar(&sequential, FlagSeq, false,
 		"sequential verification. Verify all headers sequentially as opposed to using skipping verification",
 	)
+	LightCmd.Flags().Int64Var(&startTimeout, FlagStartTimeout, 0,
+		"How many seconds to wait before staring the light client proxy. Mostly for test purposes when light client is started at the same time as the pool.")
```

Comment:
Please correct the typo: staring -> starting

Reply:
Done